webpack.config.js

```diff
@@ -70,17 +70,16 @@ module.exports = {
 
   devServer: {
     compress: true,
-    disableHostCheck: true,
+    allowedHosts: "all",
     historyApiFallback: true,
     hot: true,
-    inline: true,
+    // inline: true,
```

Comment:
Think we can just remove this line since "By default the application will be served with inline mode enabled" ? 

https://v4.webpack.js.org/configuration/dev-server/#devserverinline

Reply:
@zixiang2018 done